src/shared/components/comment/comment-node.tsx

```diff
@@ -585,14 +591,14 @@ export class CommentNode extends Component<CommentNodeProps, CommentNodeState> {
     setupTippy();
   }
 
-  handleSaveComment() {
+  async handleSaveComment() {
     this.props.onSaveComment({
       comment_id: this.commentView.comment.id,
       save: !this.commentView.saved,
     });
   }
 
-  handleBlockPerson() {
+  async handleBlockPerson() {
```

Comment:
Whats the point of making these async, they should be just pushing forms to the top level pages.

Reply:
I made them to make sure control of the form has the correct state when the browser hasn't received from the request yet. Before I did this, you had async functions being called from synchronous functions, which meant that the loading would be set to false before the request was done.